lectures/heavy_tails.md

```diff
@@ -1041,21 +1081,26 @@ np.random.seed(11)
 
 n = 120
 alphas = [1.15, 1.50, 1.75]
-
 fig, axes = plt.subplots(3, 1, figsize=(6, 8))
 
-for (a, ax) in zip(alphas, axes):
+for ax in axes:
     ax.set_ylim((-5, 50))
-    data = pareto.rvs(size=n, scale=1, b=a)
+
+for ax, alpha in zip(axes, alphas):
+    data = pareto.rvs(size=n, scale=1, b=alpha)
```

Comment:
Apologize that I do not fully understand the context, but is there a particular reason why this `for` loop needs to be separated out?